### PR TITLE
Deploy pympv on Windows

### PR DESCRIPTION
**Pull request by Luni-4**
@rr- 

These are the changes, if you will approve them I'm going to merge in them in the `master` branch

### CODE REVIEW

File: .github/workflows/deploy-windows.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: deploy-windows
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+
```
Comment (rr-) [inline .github/workflows/deploy-windows.yml]: I guess unnecessary blank line

File: .github/workflows/deploy-windows.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    runs-on: windows-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Download libmpv
+      run: |
+        $VERSION = "libmpv/mpv-dev-x86_64-20191110-git-35de8ea.7z"
+        $URL = "https://sourceforge.net/projects/mpv-player-windows/files"
```
Comment (rr-) [inline .github/workflows/deploy-windows.yml]: Is this project really maintained on sourceforge? Isn't that something that only dinosaurs and abandonware uses?

File: .github/workflows/deploy-windows.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        echo "::add-path::$VsPath"
+
+    - name: Set MSVC x86_64 linker path
+      run: |
+        $LinkGlob = "VC\Tools\MSVC\*\bin\Hostx64\x64"
+        $LinkPath = vswhere -latest -products * -find "$LinkGlob" |
+                    Select-Object -Last 1
+        echo "::add-path::$LinkPath"
+
+    - name: Create the lib
```
Comment (rr-) [inline .github/workflows/deploy-windows.yml]: `library`

File: .github/workflows/deploy-windows.yml (hunk 1 above)
Comment (Luni-4) [reply .github/workflows/deploy-windows.yml]: It helps to better discriminate the jobs if one day we are going to add another ones.

Comment (rr-) [reply .github/workflows/deploy-windows.yml]: Usually we add them at the end of the file no?

File: .github/workflows/deploy-windows.yml (hunk 2 above)
Comment (Luni-4) [reply .github/workflows/deploy-windows.yml]: So it seems. I have used the official mpv [installation](https://mpv.io/installation/) page

File: .github/workflows/deploy-windows.yml (hunk 1 above)
Comment (Luni-4) [reply .github/workflows/deploy-windows.yml]: Yep, but at least, with that blank line, we can have block of jobs clearly distinct

Comment (rr-) [reply .github/workflows/deploy-windows.yml]: It's distinguishable enough by being on top level imo but that's your call

Comment (Luni-4) [reply .github/workflows/deploy-windows.yml]: Thanks for the comprehension! :)

**Comment by Luni-4** [comment]
Also you can find the binaries produced by this action [here](https://github.com/bubblesub/pympv/releases). Feel free to test them and report back if there are issues, thanks!

**Pull request merged**